cmake/EthToolchains.cmake

```diff
@@ -1,5 +1,7 @@
 # Require C++17.
-set(CMAKE_CXX_STANDARD 17) # This requires at least CMake 3.8 to accept this C++17 flag.
+if (NOT DEFINED CMAKE_CXX_STANDARD)
```

Comment:
I just realize... apparently we don't have a consistent style for spaces for cmake files... I'm seeing a lot of cases with a space there and a lot without it...

Reply:
I think that's similar to our scripts